src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface OperationStrategy {
+    int calculate(final int number);
+}
```

Comment:
```suggestion
public interface OperationStrategy {
    OperationHandler get(Operation operation);
}
```
this service should return OperationHandler by Operation type
for BALANCE -> BalanceOperationHandler and so on

the implementation class should store handlers map - you pass it through constructor from main

Reply:
Done